CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.347 Jan 23 2024
```

Comment:
I think the changes are typically added when the PR is going to be released.

 @ciaranRoche @gdbranco how do you want to deal with this?
 
 Keeping the change in the model in one PR, and consolidate the changes in the doc when the tag is created?

Reply:
Change is good, just to not break any flow in the release process